environment/values.yaml

```diff
@@ -1,17 +1,33 @@
 labels: {}
 
+company:
+  name: #example
+  gcp:
+    enabled: false
+    bucket:
+      prefix:
```

Comment:
```
    bucket:
      prefix:
```
Don't we need a value here?

Reply:
or is this an example?

Reply:
yeah its just a variable declarartion. value for it is set in `argocd/environments/<env>/env.yaml`

Reply:
do we really need an empty variable?

Reply:
yeah i think we do. We describe an interface of our chart and this interface is used by chart users to alter its behavior. Besides that some environments do not need specific company attrs (like int for example doesnt really need bucket.prefix because it doesnt use postgres) so with this approach we can avoid setting them in `argocd/environemnts/*/env.yaml`